ssh: reset authorized_keys path to the regular one

dupremathieu opened the pull request:
The authorized_keys file path was changed to /etc/ssh/%u/authorized_keys for unknown reasons. This changed overcomplicates the configuration.

This commit resets the authorized_keys file path to the regular one ~/.ssh/authorized_keys.

dupremathieu committed:
ssh: reset authorized_keys path to the regular one

The authorized_keys file path was changed to /etc/ssh/%u/authorized_keys
for unknown reasons. This changed overcomplicates the configuration.

This commit resets the authorized_keys file path to the regular one
~/.ssh/authorized_keys.

Signed-off-by: Mathieu Dupré <mathieu.dupre@savoirfairelinux.com>

dupremathieu committed:
system-config-cluster: add a home directory for the livemigration user

Create a home directory for the livemigration user in /var/lib. This is
required to allow the user to access by SSH.

Signed-off-by: Mathieu Dupré <mathieu.dupre@savoirfairelinux.com>

TanguyRaufflet reviewed (changes requested):
You have to remove the -M option

dupremathieu committed:
system-config-cluster: unlock the livemigration user

The livemigration user needs to be unlocked if we want libvirt to be able
to use it by SSH. This is done by setting the password to '*'.

Change-Id: I0eb3717ceb5f39e38397b4a84063cde34daa7d86
Signed-off-by: Mathieu Dupré <mathieu.dupre@savoirfairelinux.com>
